hashing/src/lib.rs

```diff
@@ -44,7 +50,11 @@ pub enum Error {
 #[derive(Copy, Clone, DataSize, Ord, PartialOrd, Eq, PartialEq, Hash, Default, JsonSchema)]
 #[serde(deny_unknown_fields)]
 #[schemars(with = "String", description = "Checksummed hex-encoded hash digest.")]
-pub struct Digest(#[schemars(skip, with = "String")] [u8; Digest::LENGTH]);
+pub struct Digest(
+    #[serde(with = "ChecksummedHexForm::<[u8; Digest::LENGTH]>")]
```

Comment:
I believe @Fraser999 removed `ChecksummedHexForm` so you'll have to rewrite this...

Reply:
Reference to `ChecksummedHexForm` removed

Reply:
Tests for `serde` roundtrip and custom serialization added here: https://github.com/casper-network/casper-node/pull/2119/commits/12180009b12a68f2cda22094d9f08593e9c4ec96